Makefile

```diff
@@ -27,6 +27,17 @@ conftest:
 test: conftest
 	$(BUILD) --target test .
 
+check: check-fmt check-vet check-lint
+
+check-fmt:
+	./scripts/check-fmt.sh
```

Comment:
Adding these checks make sense to me. Looking at the script files though, could we get away with just calling `go vet ./...`, `go fmt`, etc rather than introducing files for each?

Reply:
The main reason for the scripts was ensuring the right exit codes & output was shown in the pipeline. But I can remove the scripts and use one liners for the same results.

Reply:
Sorry for the delay on this, been a buuuusy week.